openpype/modules/deadline/plugins/publish/validate_expected_and_rendered_files.py

```diff
@@ -21,6 +21,10 @@ class ValidateExpectedFiles(pyblish.api.InstancePlugin):
 
     def process(self, instance):
         self.instance = instance
+        # TODO: Find a better way to check whether a job has been submitted with
```

Comment:
Wasn't there a discussion some time ago about increasing the line character limit within OpenPype and thus tweaking the code style to allow slightly longer lines?

Reply:
I've suggested it a few times yeah but not sure if there was any traction... I think the choice of Black https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length is the best compromise

Reply:
and I have since configured my IDEs to show the 79 char line and my Black extension to use the arg to 79 chars as well but I still miss it quite often